service/frontend/wrappers/clusterredirection/api_generated.go

```diff
@@ -736,17 +743,18 @@ func (handler *clusterRedirectionHandler) RespondDecisionTaskFailed(ctx context.
 	var apiName = "RespondDecisionTaskFailed"
 	var cluster string
 
+	var token domainIDGetter
 	scope, startTime := handler.beforeCall(metrics.DCRedirectionRespondDecisionTaskFailedScope)
 	defer func() {
-		handler.afterCall(recover(), scope, startTime, cluster, &err)
+		handler.afterCall(recover(), scope, startTime, "", token.GetDomainID(), cluster, &err)
```

Comment:
`token` will be nil if it's failed to be created and returned err at 754.

Reply:
Good catch, updated the code with noopdomainIDGetter